docs/appendices/release-notes/5.7.2.rst

```diff
@@ -46,4 +46,7 @@ See the :ref:`version_5.7.0` release notes for a full list of changes in the
 Fixes
 =====
 
-None
+- Added memory accounting for multi-phase execution to prevent that a node is
+  running out-of-memory caused by in-sub-queries such as
+  ''SELECT * FROM t1 where id in (SELECT id FROM t2)'' or lookup-joins with
+  large intermediate results.
```

Comment:
```suggestion
- Added memory accounting for multi-phase execution to prevent out-of-memory
  errors caused by sub-queries such as ''SELECT * FROM t1 WHERE id IN
  (SELECT id FROM t2)'' or lookup-joins with large intermediate results.
```